src/main/java/seedu/address/logic/commands/visit/DeleteVisitsCommand.java

```diff
@@ -56,6 +56,7 @@ public String deleteVisit(Model model, List<Visit> lastShownList, LocalDate targ
                 order++;
             }
         }
+        assert order >= 1;
```

Comment:
This assert seems unnecessary since it is clear within this method code that assert has to be more than or equal to 1.

Reply:
To add on, a more meaningful assert would be one at the start of the method which asserts a value for a parameter taken in, rather than a variable created within the method itself.